src/common/components/multiSelectDropdown/multiSelectDropdown.module.scss

```
@@ -130,3 +130,15 @@
 .checkbox--isFocused {
   background-color: var(--color-fog);
 }
+
+.helpText {
+  margin: 0;
+  padding: calc(var(--spacing-m) / 2) 1rem;
```

--spacing-xs = 12px
1rem = 16px = --spacing-s
--spacing-m = 24px

```suggestion
  padding: var(--spacing-xs) var(--spacing-s);
```